src/scripts/main.js

```
@@ -355,6 +355,35 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
```

Ensure that the element with class 'dashboard' exists in the DOM before attempting to append rows to it. If the element is not found, consider logging an error or handling the situation gracefully to avoid silent failures.